### PR TITLE
Loan Product Office Name Parameter

### PR DESCRIPTION
**Pull request by anuragmath**
For Those who needs to have account number format as

Office_Name = as Stripped with Initial Letters, 
Product_short_name = concatenating both OFFICE_NAME + PRODUCT SHORT NAME

This kind of series will be useful for MFI/NBFC and Integrator, to utilize this new kind of AccountNumberGeneration

@theupscale

From Upscale Team

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/AccountNumberGenerator.java [hunk 1]
```diff
@@ -62,6 +63,7 @@ public String generate(Loan loan, AccountNumberFormat accountNumberFormat) {
         propertyMap.put(ID, loan.getId().toString());
         propertyMap.put(OFFICE_NAME, loan.getOffice().getName());
         propertyMap.put(LOAN_PRODUCT_SHORT_NAME, loan.loanProduct().getShortName());
+        propertyMap.put(OFFICE_AND_LOAD_PRODUCT_NAME, loan.getOffice().getName()+ loan.loanProduct().getShortName());
```
Comment (rajuan) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/AccountNumberGenerator.java]: This property is never read again... looks unnecessary

Comment (anuragmath) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/AccountNumberGenerator.java]: thanks for pointing out that. I have corrected it and made it necessary now

**Comment by nazeer1100126** [comment]
Changes looks OK. But this pull request is having conflicts. Can you please resolve merge conflicts and send a pull request with a single commit.

**Comment by anuragmath** [comment]
can u please tell me how can i know about the merge conflicts

**Comment by nazeer1100126** [comment]
I assume your git remote name is upstream.
Run these two commands
git fetch upstream
git rebase upstream/develop
It will get the latest code and try to merge with your changes. If merge fails, it will show the files with conflicts.

**Pull request closed without merging**